images/win/scripts/Tests/Tools.Tests.ps1

```diff
@@ -160,11 +160,10 @@ Describe "Vcpkg" {
 }
 
 Describe "VCRedist" -Skip:(Test-IsWin22) {
-    It "vcredist_2010_x64" {
-        "HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\{1D8E6291-B0D5-35EC-8441-6616F567A0F7}" | Should -Exist
-        "C:\Windows\System32\msvcr100.dll" | Should -Exist
+    It "vcredist_140" -Skip:(Test-IsWin19) {
```

Comment:
It was the test related to Windows Server 2016. Let's remove it.

Reply:
Oh yes now I understand, the whole group is skipped for Windows 2022, and this one for Windows 2019 - make sense indeed!